applications/FluidDynamicsApplication/tests/buoyancy_test.py

```diff
@@ -83,7 +81,8 @@ def setUpModel(self):
         thermal_settings.SetVelocityVariable(VELOCITY)
         thermal_settings.SetMeshVelocityVariable(MESH_VELOCITY)
         if self.convection_diffusion_solver == 'bfecc':
-            thermal_settings.SetProjectionVariable(PROJECTED_SCALAR1)
+            import KratosMultiphysics.ConvectionDiffusionApplication as ConvDiff
```

Comment:
Why don't we import it in the top of the file?

Reply:
For no good reason, really. I moved it to the top for clarity.